tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -25,32 +26,33 @@
             *[constants.CEPHBLOCKPOOL, '1', '600M'], marks=pytest.mark.polarion_id('OCS-2356')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '25', '15GB'], marks=pytest.mark.polarion_id('OCS-2340')
+            *[constants.CEPHBLOCKPOOL, '25', '15G'], marks=pytest.mark.polarion_id('OCS-2340')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '50', '30GB'], marks=pytest.mark.polarion_id('OCS-2357')
+            *[constants.CEPHBLOCKPOOL, '50', '30G'], marks=pytest.mark.polarion_id('OCS-2357')
         ),
         pytest.param(
-            *[constants.CEPHBLOCKPOOL, '100', '60GB'], marks=pytest.mark.polarion_id('OCS-2358')
+            *[constants.CEPHBLOCKPOOL, '100', '60G'], marks=pytest.mark.polarion_id('OCS-2358')
         ),
         pytest.param(
             *[constants.CEPHFILESYSTEM, '1', '600M'], marks=pytest.mark.polarion_id('2341')
         ),
         pytest.param(
-            *[constants.CEPHFILESYSTEM, '25', '15GB'], marks=pytest.mark.polarion_id('2355')
+            *[constants.CEPHFILESYSTEM, '25', '15M'], marks=pytest.mark.polarion_id('2355')
```

Comment:
Isn't it 15G?

Reply:
@ramkiperiy - fillup_fs receives as input either M or GB units.

Reply:
Not that, I meant the value as per the list of parameters I see for CEPHBLOCKPOOL interface pvc_size values are "600Mi, 15Gi, 30Gi, 60Gi" and for CEPHFILESYSTEM interface pvc_size values are  "600Mi, 15Mi, 30Gi, 60Gi". In CEPHFILESYSTEM interface there is 15Mi, just want to make sure either intention is to create 15Mi or 15Gi.

Reply:
@ramkiperiy  - you are right! Caught a typo! It should be of course 15Gi! Fixing it.